guides/v2.0/coding-standards/technical-guidelines.md

```diff
@@ -525,7 +527,81 @@ class View extends Template
 
 ### 6.4. Service Contracts (Application) layer
 
-We are reviewing this section and will publish it soon.
+6.4.1. Location of API interfaces
+
+6.4.1.1. Service contract interfaces SHOULD be placed in separate API modules. The other modules will depend on the API module, and implementations could be easily swapped via `di.xml`. API module namess must end with the `Api` suffix. For example, if a module is named `MyModule`, its APIs SHOULD be declared in a module named `MyModuleApi`.
```

Comment:
> API module names**s** _must_ end with the `Api` suffix

I think that there should be "SHOULD" not "must"

Reply:
What is the case when we don't want this?